dev-requirements.txt

```diff
@@ -1,3 +1,19 @@
-pytest==2.8.5
-pytest-mock==0.9.0
-responses
+#
+# This file is autogenerated by pip-compile
+# To update, run:
```

Comment:
Updating in this case means that all the pinning will move forward to the latest versions, right?  How often would you expect to do that?